lighthouse-core/lib/tracehouse/trace-processor.js

```diff
@@ -85,8 +85,8 @@ class TraceProcessor {
    */
   static _isNavigationStartOfInterest(event) {
     return event.name === 'navigationStart' &&
-      (!event.args.data || !event.args.data.documentLoaderURL ||
```

Comment:
would you be fine with going longhand on this?

this is def an improvement over `&& (!foo || !bar)`, but imo the multiple return version i put in chat is even friendlier.